### ISSUE TITLE
List of branches incomplete if there are too many branches

### ISSUE THREAD

**Issue by mattrussell-sonocent** [ccmenu/ccmenu2#18]
Version 26.0 (244364B)

In the branch selector menu for a repo I work on, there are so many branches the options are truncated, and the `main` branch can't be selected - and it doesn't appear to be possible to select it any other way through the UI.

<img width="458" alt="image" src="https://github.com/user-attachments/assets/c386fda0-6d77-4efc-8c86-87929cf54da0">

As a workaround, I selected a different branch, exported pipelines, edited the JSON, and imported the file again.

**Comment by erikdoe** [comment]
Yes, this is a known problem. Of course CCMenu could fetch all branches, or the user could actively say "fetch more", but then a pop-up isn't a great UI for that. I wanted to use a search-box with drop-down but that UI element doesn't exist in Swift UI and it's proved to be quite difficult to implement it. Will return to this sooner or later.
